src/routes/index.go

```diff
@@ -1,11 +1,20 @@
 package routes
 
 import (
-	"github.com/Real-Dev-Squad/gopher-cloud-service/src/controllers"
+	"strconv"
+
 	"github.com/gin-gonic/gin"
+
+	"github.com/Real-Dev-Squad/gopher-cloud-service/src/config"
 )
 
-func UseRoutes(router *gin.Engine, version string) {
-	ver := "/v" + version
-	router.GET(ver+"/health", controllers.Health)
+func UseRoutes(router *gin.Engine) {
+	ver := "/v" + strconv.Itoa(config.Global.Version)
+	indexGroup := router.Group(ver)
+	{
+		healthGroup := indexGroup.Group("/health")
+		{
+			HealthRoutes(healthGroup)
+		}
+	}

```

Comment:
What does this block do?

Reply:
it groups the routes so that the indexGroup specifies "/v1" so that anything inside this block wont have to use the /v1 in them inner apis can directly use /health or /users

Reply:
Also, a good catch removed the curly braces